src/main/java/github/grace000/erversay/request/RequestRouter.java

```diff
@@ -0,0 +1,21 @@
+package github.grace000.erversay.request;
+
+import java.util.Hashtable;
+import static github.grace000.erversay.Constants.*;
+
+public class RequestRouter {
+    private Hashtable<String, String> routeTable = new Hashtable<>();
```

Comment:
Any reason you used `Hashtable` instead of `Hashmap`?

Reply:
After further research I realized that I shouldn't be using Hashtable. I didn't realize that they both provided key value pairs. Thanks!